crates/astria-conductor/src/driver.rs

```diff
@@ -63,7 +69,9 @@ pub struct Driver {
     /// The channel used to send messages to the executor task.
     executor_tx: executor::Sender,
 
-    network: GossipNetwork,
+    network: SyncWrapper<GossipNetwork>,
```

Comment:
Can you add comments on a) why this is necessary, and b) link to the rust internals discussion and the `Exclusive`type that is still unstable?